aggregates/account/src/main/java/com/bloxbean/cardano/yaci/store/account/storage/impl/AccountBalanceStorageImpl.java

```diff
@@ -153,6 +160,37 @@ private void saveAddrBalanceBatchJOOQ(List<AddressBalanceEntity> addressBalanceE
 
     }
 
+    private void saveAddressBatch(List<AddressEntity> addressEntities) {
+        if (enableJPAInsert) {
+            //TODO
+        } else {
+            if (log.isTraceEnabled())
+                log.trace("Inserting address using JOOQ batch");
+
+            transactionTemplate.execute(status -> {
```

Comment:
Probably we don't need a new transaction for address saving as # of addresses should not be too much.